### ISSUE TITLE
Program crashes upon starting

### ISSUE THREAD

**Issue by Reismann69** [MiningMark48/Minecraft-JSON-Generator#5]
Whenever I run the program, it just pops up a window for a split second and then closes. There's no sign of activity in the task manager either, so it looks like the app just crashes straight away. (using windows10)

**Comment by Artex154** [comment]
Hello, this is a virus, so it is normal its not opening.

**Comment by Reismann69** [comment]
> Hello, this is a virus, so it is normal its not opening.

Whatt???? How am I gonna get rid of it? 😥

**Comment by Artex154** [comment]
Its a trojan i think you have to factory reset windows

**Comment by SiriosDev** [comment]
Source? 😂

**Comment by Artex154** [comment]
TotalVirus, the jar seems safe, but the .exe, is a 5/70

![Image](https://github.com/user-attachments/assets/78262923-d670-4cef-8d34-22f240276e13)

**Comment by SiriosDev** [comment]
Or perhaps, since it has not been updated since 2017, it is simply too old to follow the correct safety guidelines

**Comment by Artex154** [comment]
No, a trojan is a trojan, and otherwise it would be 1 or 2 not 5.

**Comment by SiriosDev** [comment]
Sorry but 7.14% doesn't exactly sound like enough to say “_it's a trojan_” it's more like _“there's a small chance it's a trojan_”, by the way in those 5 there are 1/2 of them with a good reputation, the other 3/4 are Mr. nobodies

also one of the causes of alert would be the simply expired site that now no longer hosts the creator's site but something Chinese

**Comment by Artex154** [comment]
Do what you think. I believe its a trojan because i trust virustotal.

![Image](https://github.com/user-attachments/assets/b8289dc5-2169-4455-ba31-a02d1f849c7e)

**Comment by SiriosDev** [comment]
and in fact virus total tice that 5 “guys” out of 70 find something wrong with it, of which the only ones with a minimally reliable name are acronis and malwerbytes, or rather I misread not even acronis, only malwerbytes, while all the others don't think there is anything wrong with it

**Comment by SiriosDev** [comment]
reading only the summary instead of the whole report and from there making disinformation, is not exactly the healthiest thing in the universe
